src/whylogs/app/logger.py

```diff
@@ -75,45 +80,82 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
 
+
+    @property
+    def profile(self,):
+        """
+        :return: the last backing dataset profile
+        :rtype: DatasetProfile
+        """
+        return self._profiles[-1]["full_profile"]
```

Comment:
what does the last backing DS represent here? A segment or  the full dataset? Do we keep all of the profiles in memory?

Reply:
It is the current active profile that is updated by l`ogger.log_*`.